test/widgets/after_registration_page_test.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:motis_mitfahr_app/welcome/pages/after_registration_page.dart';
+import 'package:motis_mitfahr_app/welcome/pages/login_page.dart';
+import 'package:motis_mitfahr_app/welcome/pages/welcome_page.dart';
+
+import '../util/pump_material.dart';
+
+void main() {
+  setUpAll(() async {
+    SupabaseManager.setCurrentProfile(null);
+  });
+
+  group('WelcomePage', () {
+    testWidgets('Shows the page', (WidgetTester tester) async {
```

Comment:
is this test really necessary or did you mean `'Shows the image'`.

Reply:
Now I just want to test that it shows _something_ and does not crash. The page is literally just an image and some text

Reply:
ah okay i understand.